### ISSUE TITLE
ValueError in FastFit.make_measurment

### ISSUE THREAD

**Issue by bednya** [flav-io/flavio#69]
The code (from https://github.com/DavidMStraub/flavio-tutorial/blob/master/4%20Fits.ipynb)
```python
import flavio
from flavio.statistics import fits
fit = fits.FastFit(name="My First Fast Fit",
                   fit_parameters=['Vcb'],
                   observables=['BR(B+->Denu)', 'BR(B+->Dmunu)'])

%time fit.make_measurement(N=100, force=True)                    
```
produces the following output with  flavio v.1.0:

    ---------------------------------------------------------------------------

    ValueError                                Traceback (most recent call last)

    ~/.local/lib/python3.5/site-packages/flavio/statistics/fits.py in make_measurement(self, N, Nexp, threads, force, force_exp)
        714         if np.asarray(central_exp).ndim == 0 or len(central_exp) <= 1: # for a 1D (or 0D) array
        715             m.add_constraint(self.observables,
    --> 716                     NormalDistribution(central_exp, np.sqrt(covariance)))
        717         else:
        718             m.add_constraint(self.observables,

    ~/.local/lib/python3.5/site-packages/flavio/statistics/probability.py in __init__(self, central_value, standard_deviation)
        225                          support=(central_value - 6 * standard_deviation,
        226                                   central_value + 6 * standard_deviation))
    --> 227         if standard_deviation <= 0:
        228             raise ValueError("Standard deviation must be positive number")
        229         self.standard_deviation = standard_deviation

    ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

```python

**Comment by DavidMStraub** [comment]
Oh. I think this bug was introduced by this change:
https://github.com/flav-io/flavio/commit/953599969bd4a2d33d1d326342f0b0094b957775#diff-9c0dee82300a06c187aa587fb4c108abR512

The problem here is that `means` is a list of arrays in general, in this case a list with a single lenght-2 array, because there is a single measurement instance but it constraints 2 fit observables. The change from `means[0]` to `means[0][0]` picks out only the first one, and then things go wrong.

@peterstangl, what was the reason for this change? It was just about returning a number rather than a length-1 array, right? Can that be reverted or do we have to insert an additional check for the length of `means[0]`?

**Comment by peterstangl** [comment]
@DavidMStraub, yes the reason was to return a number instead of a length-1 array. Otherwise, I think the final likelihood was also a length-1 array instead of a number. I thought the right place to address this issue is there. But I was clearly missing the case where there is only a single measurement that actually constraints 2 fit observables.
So I would suggest to either add a check for the length of `means[0]`, or to revert my modification and then address the issue with the length-1 array somewhere else (but I don't know where this should ideally be done).
